policy-build-go/action.yaml

```
@@ -14,7 +14,7 @@ inputs:
     default: policy.wasm
   tinygo-version:
     required: true
-    default: 0.18.0
+    default: 0.20.0
```

```suggestion
    default: 0.24.0
```

We need at least 0.21 to be able to use the upstream version of `gjson`. Our [Go template project](https://github.com/kubewarden/go-policy-template/blob/main/Makefile) is already using 0.23.0.

I propose to jump to 0.24 which has just been released.

---

Okay. I'll update the version

---

Done! :)